### PR TITLE
Refactor labkey-api-java repository

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
The extra `labkey-client-api` folder in this repository is unnecessary, left over from when the repo used to house `labkey-api-sas` as well.

#### Related Pull Requests
* https://github.com/LabKey/labkey-api-java/pull/34

#### Changes
* Eliminate unnecessary intermediate folder

### CODE REVIEW

**Comment by labkey-adam** [comment]
@labkey-tchad I think I'll need your help to get the Java Remote API suite to run after this refactor

File: gradle.properties [hunk 1]
```diff
@@ -7,14 +7,14 @@ artifactory_contextUrl=https://labkey.jfrog.io/artifactory
 sourceCompatibility=17
 targetCompatibility=17
 
-gradlePluginsVersion=1.36.1
+gradlePluginsVersion=1.38.0
```
Comment (labkey-susanh) [inline gradle.properties]: We're going to need a new gradle plugin release for this change. We have a method there called `getRemoteApiProjectPath` that is used for declaring dependencies in various places

**Review by labkey-susanh — COMMENTED** [review]
For better or for worse, we currently publish this under org.labkey.api/labkey-client-api This will no longer be the case by default with this move. We'll need to either update the repository name to be labkey-client-api or change the name in build.gradle file.

**Comment by labkey-tchad** [comment]
> For better or for worse, we currently publish this under org.labkey.api/labkey-client-api This will no longer be the case by default with this move. We'll need to either update the repository name to be labkey-client-api or change the name in build.gradle file.

Whatever the name we chose, we can't rely on the repository name to define it. TeamCity uses some internal logic to name the root directory for builds. This causes gradle to produce some real nonsense names:
image

@labkey-adam The solution is to define the name in `settings.gradle`
```
rootProject.name='labkey-client-api' # or 'labkey-api-java'
```

**Comment by labkey-adam** [comment]
@labkey-tchad of course, 77414112b699049a... not sure why I couldn't find my artifact. :) https://labkey.jfrog.io/ui/packages/gav:%2F%2Forg.labkey.api:77414112b699049a?name=77414112b699049a&type=packages

**Review by labkey-susanh — APPROVED** [review]

**Pull request merged**